Changed font to Avenir Next

rahulbisht12 opened the pull request:
Fixes #20

AkankshaBodhankar commented:
Can you post a screenshot here ?

AkankshaBodhankar commented on css files/gethelpnow-style.css:
```diff
@@ -61,7 +75,7 @@ body {
     float: right;
     font-size: 28px;
-    font-weight: bold;
+	font-family: 'Avenir Next Bold';
```
remove extra spacing

AkankshaBodhankar commented:
code looks good! Just adjust the spacing and resolve the conflicts. I would suggest it's better to make a new PR with the conflicts resolved and changes made. Once you do this you will be done with the task

rahulbisht12 commented:
@AkankshaBodhankar I have updated the PR and squashed all the commits so its now easy to merge the changes. Here are the screenshots. :)

![p2](https://cloud.githubusercontent.com/assets/16523310/20794346/de883a0e-b7f1-11e6-981f-e065d47d42b7.png)
![p3](https://cloud.githubusercontent.com/assets/16523310/20794345/de84dd00-b7f1-11e6-8133-6a02f7f5433a.png)

rahulbisht12 commented:
@AkankshaBodhankar Why the "Travis CI build failed" for this commit ?

AkankshaBodhankar commented:
That's an issue from our end don't worry about it. You work looks good. Keep it up !

rahulbisht12 commented:
@AkankshaBodhankar When this PR will be merged ?

AkankshaBodhankar commented:
Right now we are still reviewing the PR's so, merging won't be done immediately

dannymout commented:
Question: Do you have the license for using Avenir Next in an app? Otherwise, it could lead to legal problems with whoever owns the rights to it. I highly suggest going with a free font, [Google Fonts](fonts.google.com) has a selection of high-quality fonts that are free. Also, an interesting approach I like to use is writing the CSS to use the main system font, ex. (macOS/iOS: San Francisco, Android/ChromeOS: Roboto, Windows: Segoe UI)

This code allows you to do so: `font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
`

AkankshaBodhankar commented:
@dannymout Good suggestion! We can use free fonts instead. PR has already been made for this issue. Once it is merged feel free to make the changes.
